### ISSUE TITLE
stg mail send with multiple --cc options only honors the first

### ISSUE THREAD

**Issue by chaudron** [stacked-git/stgit#460]
When I use the following command:

```
stg email send --to dev@openvswitch.org --annotate ./patch --cc aaaa@ovn.org --cc bbb@redhat.com --cc ccc@redhat.com
```

Only the first --cc is included in the sent email.

**Comment by jpgrayson** [comment]
Hi @chaudron. Thanks for writing this up.

However, am not able to reproduce this issue. I modified an existing test case in `t1903-email-send.sh` to add several `--cc` options and I am able to observe all of them in the `Cc:` header.

The tests I ran use `--dry-run`. I wonder if that might be making a difference?

I also wonder what version of git you were running? I tested with the latest 2.45.2.

`stg email send` is just a thin wrapper on `git send-email`, so I'm not sure what StGit could do to cause this kind of breakage. If you run `git send-email` directly, does the problem still occur?
